index.html

```diff
@@ -1,52 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-    <title>Portfolio</title>
-</head>
-<body>
-  <section class="first-page">
-    <div class="main">
-      <nav class="navbar">
-        <div class="logo"><a href="#">Eros</a></div>
-        <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
-    </nav>
-    <div class="headline">
-    <div class="description">
-      <h1 class="name"> I am Aime
-          Glad to see you!</h1>
-         
-      <p class="text"> I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-           </p>
-      </div>
-      <div class="contact">
-        <button>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"rel="stylesheet"/>
+    <link rel="stylesheet" href="styles.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"rel="stylesheet"/>
 
-          <h2 class="media">LET'S CONNECT:</h2>
-        
-        </button>
-        <ul>
-         
-          <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
+    <title>Porfolio</title>
+  </head>
+  <body>
+    <header class="mobile-portfolio">
+      <a href="#" class="logo">
+        <h2>Eros</h2>
+      </a>
+      <nav class="menu"><img src="./image/Union.svg" alt=""></nav>
```

Comment:
- [ ] I think while  making the suggested changes by the previous reviewer, you removed the icon, Kindly make sure the ```hamburger``` icon is visible on your page with the same color as the Figma design 😎

### Your design
![image](https://user-images.githubusercontent.com/102757126/212367524-f223dc0d-39c6-493a-bd37-13c4ed8fbee7.png)